lib/matplotlib/axis.py

```diff
@@ -1317,6 +1316,9 @@ def get_minorticklocs(self):
         # Remove minor ticks duplicating major ticks.
         major_locs = self.major.locator()
         minor_locs = self.minor.locator()
+        # we do check when we set the attribute that this is a Locator
+        # subclass, use getattr out of an over abundance of caution
+        remove_overlaps = getattr(self.minor.locator, 'remove_overlaps', True)
```

Comment:
OK, so you went w/ the attribute path?   So, how does this get set?  I guess this PR needs tests and examples...